main.c

```diff
@@ -132,10 +136,16 @@ bool start_mp(safe_mode_t safe_mode) {
         serial_write(MSG_SAFE_MODE_NO_MAIN);
     } else {
         new_status_color(MAIN_RUNNING);
-        found_main = maybe_run("code.txt", &result) ||
-                     maybe_run("code.py", &result) ||
-                     maybe_run("main.py", &result) ||
-                     maybe_run("main.txt", &result);
+        const char *supported_filenames[] = {"code.txt", "code.py", "main.py", "main.txt",""};
```

Comment:
How about adding a macro to do this initialization? That way it can add the empty string.

`STRING_LIST("code.txt", "code.py", "main.py", "main.txt")`

Reply:
Will do.

Reply:
I don't see a place for similar macros like `supervisor/default/messages.h` is for message strings. Should I put it in `main.c` or somewhere else?

Reply:
In this file is fine. I'd probably just put it above maybe_run_list.